tox.ini

```diff
@@ -139,6 +139,9 @@ deps =
     sanic-0.8: sanic>=0.8,<0.9
     sanic-18: sanic>=18.0,<19.0
     sanic-19: sanic>=19.0,<20.0
+    sanic-20: sanic>=20.0,<21.0
+    {py3.7,py3.8,py3.9}-sanic-21: sanic>=21.0
```

Comment:
You haven't declared those jobs to run in line 40. If you run `tox -l` there will not be an entry for sanic 20 and sanic 21.